workbench/src/renderer/app.jsx

```diff
@@ -121,7 +120,9 @@ export default class App extends React.Component {
    * @param {InvestJob} job - as constructed by new InvestJob()
    */
   openInvestModel(job) {
-    const navID = crypto.randomBytes(16).toString('hex');
+    const navID = window.crypto.getRandomValues(
```

Comment:
Ahh there's actually two unrelated crypto APIs at play. This one at `window.crypto` is a built-in browser API (nice!). 

The other one used elsewhere (to do some hashing in `InvestJob.js` I think) is the node module, which has a lot more functionality. I guess we had a use-case for that node module in the renderer, so that's why it's in preload. But right here in `app.jsx` it looks like the `window.crypto` was all we needed to get some random values.

Reply:
Thanks for clarifying that!